webapp/src/components/SampleTablePrime.vue

```diff
@@ -10,7 +10,7 @@
       selection-mode="multiple"
       :value="samples"
       paginator
-      :rows="10"
+      :rows="20"
       :rows-per-page-options="[10, 20, 50, 100]"
```

Comment:
Hey @jdbocarsly, as requested yesterday, this is where you can easily modify the number of items per row, or the different choices offered in the dropdown. Don't hesitate to let me know if you want me to change anything.